Warning in Polynomial test with LEDA

## Issue Details

See [here](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.0-Ic-71/Algebraic_kernel_d/TestReport_lrineau_Fedora-with-LEDA.gz) : 

```
/mnt/testsuite/include/CGAL/Test/_test_polynomial_traits_d.h:1861:59: warning: typedef ‘PT_Integer_4’ locally defined but not used [-Wunused-local-typedefs]
```
## Source Code
```
#ifdef CGAL_USE_LEDA
{        
  typedef CGAL::LEDA_arithmetic_kernel AT;
  typedef typename AT::Integer Integer; 
  typedef typename AT::Rational Rational;
  const int dimension = 4;
  typedef typename PT:: template Rebind<Integer,4>::Other PT_Integer_4;
  typedef typename PT:: template Rebind<Rational,4>::Other PT_Rational_4;
  CGAL_static_assertion((boost::is_same< typename PT_Integer_4::Innermost_coefficient_type, 
          Integer>::value));
  CGAL_static_assertion((boost::is_same< typename PT_Rational_4::Innermost_coefficient_type, 
          Rational>::value));
  CGAL_static_assertion((PT_Integer_4::d==dimension));
  CGAL_static_assertion((PT_Rational_4::d==dimension));
}
#endif
```
## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Fedora with LEDA

--- a/Polynomial/include/CGAL/Test/_test_polynomial_traits_d.h
+++ b/Polynomial/include/CGAL/Test/_test_polynomial_traits_d.h
@@ -1860,7 +1860,9 @@ void test_rebind(const PT& /*traits*/){
   typedef typename AT::Rational Rational;
   const int dimension = 4;
   typedef typename PT:: template Rebind<Integer,4>::Other PT_Integer_4;
+  CGAL_USE_TYPE(PT_Integer_4);
   typedef typename PT:: template Rebind<Rational,4>::Other PT_Rational_4;
+  CGAL_USE_TYPE(PT_Rational_4);
   CGAL_static_assertion((boost::is_same< typename PT_Integer_4::Innermost_coefficient_type, 
           Integer>::value));
   CGAL_static_assertion((boost::is_same< typename PT_Rational_4::Innermost_coefficient_type, 

